cmd/host-profiler/command/command.go

```diff
@@ -0,0 +1,52 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build hostprofiler
```

Comment:
Do we actually need a build tag ?
All the code in `cmd/host-profiler` should only be used by that binary, so no risk of adding dependencies in other builds (?)

Reply:
If we do need a build tag, then it should be added in the list in `tasks/build_tags.py`

Reply:
The main reason why I added build tag is that I have to add a **temporary** dependency to https://github.com/datadog/dd-otel-host-profiler which uses datadog-agent repository. This will create a cycle between these 2 repositories and I would like to avoid bothering other developers. 
I have to think deeper to know if using build tag really help solving this problem.

Reply:
Good catch. I think the reason is that we import dd-trace-go, and dd-trace-go relies on agent components. Does this mean that we won't be able to enable profiling through dd-trace-go for debugging in the agent repo?